R/plot_functions.R

```diff
@@ -245,10 +245,10 @@ plot_krona<-function(physeq,output,variable, trim=F){
   # Abundance and taxonomic assignations for each OTU are fetched
   # and written to a file that would be processed by Krona.
   for( lvl in levels(df[,2])){
-    write.table(
-      unique(
-        df[ which( df[,2] == lvl), -2]
-        ),
+      tt = df[which(df[, 2] == lvl), -2]
+      tt2 = tt[which(tt[,1]!=0),]
+
+    write.table(tt2,
```

Comment:
Well spotted. I think I did that initially on the toy dataset I used. But it does not make sense with real datasets.

Reply:
Thanks for your reply, spotted this by using this function with big real dataset.